pkg/pcrbruteforcer/reproduce_expected_pcr0_test.go

```diff
@@ -113,48 +127,60 @@ func BenchmarkReproduceExpectedPCR0(b *testing.B) {
 			}
 			measurements, _, _, err := pcr.GetMeasurements(firmware, 0, measureOptions...)
 			if err != nil {
-				b.Fatal(err)
+				panic(err)
 			}
 
 			b.Run("correctPCR0", func(b *testing.B) {
 				b.ReportAllocs()
 				b.ResetTimer()
 				for i := 0; i < b.N; i++ {
-					_, _, _, _ = ReproduceExpectedPCR0(
+					_, err := ReproduceExpectedPCR0(
 						ctx,
 						pcr0Correct,
 						pcr.FlowIntelCBnT0T,
 						measurements,
 						firmware.Buf(),
+						settings,
 					)
+					if err != nil {
+						panic(err)
+					}
 				}
 			})
 
 			b.Run("incompletePCR0", func(b *testing.B) {
 				b.ReportAllocs()
 				b.ResetTimer()
 				for i := 0; i < b.N; i++ {
-					_, _, _, _ = ReproduceExpectedPCR0(
+					_, err := ReproduceExpectedPCR0(
 						ctx,
 						pcr0Incomplete,
 						pcr.FlowIntelCBnT0T,
 						measurements,
 						firmware.Buf(),
+						settings,
 					)
+					if err != nil {
+						panic(err)
+					}
 				}
 			})
 
 			b.Run("invalidPCR0", func(b *testing.B) {
 				b.ReportAllocs()
 				b.ResetTimer()
 				for i := 0; i < b.N; i++ {
-					_, _, _, _ = ReproduceExpectedPCR0(
+					_, err := ReproduceExpectedPCR0(
 						ctx,
 						pcr0Invalid,
 						pcr.FlowIntelCBnT0T,
 						measurements,
 						firmware.Buf(),
+						settings,
 					)
+					if err != nil {
+						panic(err)
```

Comment:
Oh, well imagine
As I'm pacing the pews in a church corridor
And I can't help but to hear
No, I can't help but to hear an exchanging of words

Reply:
Fixed.